StudentAttendancePortal/app/mailers/teacher_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class TeacherMailer < ApplicationMailer
+  default from: 'julee.gunjal@joshsoftware.com'
+
+  def welcome_email
+    puts "INNNNN mailer"
+    puts params
+    @teacher = params[:teacher]
+    # @url  = 'http://gmail.com/login'
+    mail(to: @teacher.email, subject: 'You are successfully registered on Student Attendance portal')
```

Comment:
Try to move all statement in local.
Ref https://guides.rubyonrails.org/i18n.html